### ISSUE TITLE
Warning: docstring has wrong usage of unescaped single quote

### ISSUE THREAD

**Issue by ideasman42** [riscy/melpazoid#24]
Warning when updating melpazoid

With emacs-30 (build from master), I'm getting warnings.

```
In melpazoid--package-load-paths:
.config/emacs/local/melpazoid.el:58:2: Warning: docstring has wrong usage of unescaped single quotes (use \=' or different quoting such as `...')

In melpazoid--buffer-almost-empty-p:
.config/emacs/local/melpazoid.el:70:2: Warning: docstring has wrong usage of unescaped single quotes (use \=' or different quoting such as `...')
```

**Comment by riscy** [comment]
Thanks for raising this -- and I apologize for not seeing this sooner.  This is interesting that single quotes aren't allowed in docstrings (or, at least, that's the proposal).  I see exactly where this is happening, but I'll hold off until Emacs 30 lands.

**Comment by riscy** [comment]
Likely fixed with 719ce96 - will close.
